tests/conftest.py

```diff
@@ -221,22 +223,26 @@ def mxnet_eia_latest_py_version():
 
 @pytest.fixture(scope="module", params=["py2", "py3"])
 def pytorch_training_py_version(pytorch_training_version, request):
-    if Version(pytorch_training_version) < Version("1.5.0"):
-        return request.param
+    if Version(pytorch_training_version) >= Version("1.13"):
+        return "py39"
     elif Version(pytorch_training_version) >= Version("1.9"):
         return "py38"
-    else:
+    elif Version(pytorch_training_version) >= Version("1.5"):
         return "py3"
+    else:
+        return request.param
 
 
 @pytest.fixture(scope="module", params=["py2", "py3"])
 def pytorch_inference_py_version(pytorch_inference_version, request):
-    if Version(pytorch_inference_version) < Version("1.4.0"):
-        return request.param
+    if Version(pytorch_inference_version) >= Version("1.13"):
```

Comment:
Same applies to this function _pytorch_inference_py_version_ as above:

Reply:
should be 1.4.0 for inference. 1.5.0 for training. otherwise the new logic is fine.

Reply:
thanks for catching this!